wiki/Internet_Relay_Chat/en.md

```diff
@@ -1,100 +1,84 @@
+- **Server:** `cho.ppy.sh` or `irc.ppy.sh`
+- **Port:** `6667` (default)
+- **Username:** Your osu! username (replace spaces to underscores)
+- **Password:** Get it at [IRC Authentication](https://osu.ppy.sh/p/irc).
 
-`* Welcome to osu!bancho.`
-`* -`
-`* - You are required to authenticate before accessing this service.`
-`* - Please click the following link to complete this process:`
+*Your IRC password is different from your account password. **Do not share it to others**.*
 
-When you go to the URL you are presented with you'll see a screen with a big fancy "Authorise IRC connection" button. Simply clicking this will allow you into the chat and auto-join you into [\#osu](irc://cho.ppy.sh/osu).
-
-If you wish to not have to click the big fancy button every time you connect you can put the password given into either your server's password field in a profile or use it while connecting.
-
-`To permanently authorise a client, please change your IRC password (server password) to: XXXXXXX`
-
-Remember **NOT TO GIVE YOUR IRC PASSWORD** to anyone else.
 
 Basic IRC Commands
 ==================
+| Command              | Description                           |
+| -------------------- | ------------------------------------- |
+| `/join <#channel>`   | Joins a channel                       |
+| `/part <#channel>`   | Leaves a channel                      |
+| `/me <action>`       | Sends an action message               |
+| `/ignore <username>` | Ignores a user (hides their messages) |
 
-| Description                 | Command             |
-|-----------------------------|---------------------|
-| Join channel (I.E. \#lobby) | /join \#somechannel |
-| Leave channel               | /part               |
-| Ignore nick                 | /ignore somenick    |
-| Make actions                | /me does something  |
 
 Disabling Join/Part Messages
 ============================
-
-Every time someone joins a channel or leaves it a message like this is shown:
-
-`someuser has joined #somechannel`
-`someuser has quit #somechannel`
-
-While in low traffic channels this is usually isn't disruptive, people are joining and leaving \#osu constantly making it potentially hard to keep track of the chat.
+Knowing players who joins or parts is a good thing, however in very crowded such as #osu, you will constantly receive join/part messages and will not be able to keep track of conversations. Therefore, these must be disabled.
+
+```
+[17:46] * lauripihl (cho@ppy.sh) has left #lobby
+[17:46] * Kastun (cho@ppy.sh) has joined #lobby
+[17:46] * AuReL (cho@ppy.sh) has joined #lobby
+[17:46] * osukd (cho@ppy.sh) has joined #lobby
+[17:46] * BreadTooGood (cho@ppy.sh) has joined #lobby
+[17:46] * keanyew18 (cho@ppy.sh) has joined #lobby
+[17:46] * JaKox (cho@ppy.sh) has joined #lobby
+[17:46] * Kerantor (cho@ppy.sh) has joined #lobby
+```
 
 Disabling Join/Part Messages In Common Clients
 ----------------------------------------------
```

Comment:
If you do change the heading levels for `## Disabling Join/Part Messages`, then I assume that this one should also go down a level
```
### Disabling Join/Part Messages In Common Clients
```